plutus-tutorial/doc/intro.adoc

```diff
@@ -29,3 +29,14 @@ authors do not write Plutus Core directly. The Plutus Platform is a
 software development kit to enable smart contract authors to easily
 write smart contracts, including the logic that will eventually be run
 on the blockchain as Plutus Core.
+
+To code a smart contract in Plutus, you can code directly in the Plutus 
+Playground editor, or you can use your preferred code editor and save the 
+files in Haskell format (.hs) to get the syntax highlighting, 
```

Comment:
Use code formatting for `.hs`.

Reply:
do you mean replace this sentence? unsure as to what that means

Reply:
will do